src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -19,6 +19,157 @@ namespace processor {
 
 ParsingDriver::ParsingDriver(common::DataChunk& chunk) : chunk(chunk), rowEmpty(false) {}
 
+void copyStringToVector(ValueVector* vector, uint64_t rowToAdd, std::string_view strVal,
```

Comment:
I guess you can remove this function?

Reply:
cast to struct is still using this function